integration-tests/pom.xml

```diff
@@ -435,7 +435,6 @@
                 <docker.build.skip>false</docker.build.skip>
                 <docker.build.hadoop>false</docker.build.hadoop>
                 <it.indexer>middleManager</it.indexer>
-                <zk.version>3.5</zk.version>
```

Comment:
it should be removed from .travis.yml too?

Reply:
Good question. I think it should be, but I'm unable to validate this change on my local so I leave it there. Will Any change to .travis.yml  be reflected on the CI in this PR ? If it's, I can push a commit to delete the default version in that file.

Reply:
> Will Any change to .travis.yml be reflected on the CI in this PR ?

Yes. It will.